src/lib/FieldType/DataTransformer/SingleCountryValueTransformer.php

```diff
@@ -36,6 +36,10 @@ public function transform($value)
             return null;
         }
 
+        if (empty($value->countries)) {
```

Comment:
Should we not also check if key exist in cases like this?

Reply:
I decided to throw TransformationFailedException if `Alpha2` key doesn't exist.